fk/ui.go

```diff
@@ -51,3 +52,17 @@ func printFailedAction(message string) {
 func printHeader(message string) {
 	out.Print(colour.Header(fmt.Sprintf(" %-79s", message)) + "\n\n")
 }
+
+func printFileDivider(message string) {
+	if message != "" {
+		message = " " + message + " "
+	}
+	leftHandAndMessage := strings.Repeat(fileDividerRune, 2) + colour.File(message)
+	rightHand := strings.Repeat(fileDividerRune, (fileDividerLength - len(leftHandAndMessage)))
```

Comment:
Ooh I would have expected `len(leftHandMessage)` to include the length of the `colour.File(..)` control bytes, is that not the case

Reply:
I don't follow? I think it does?